composable_kernel/include/tensor_description/tensor_descriptor.hpp

```diff
@@ -69,7 +69,7 @@ struct TensorDescriptor
             Number<ndim_visible_>{});
 
         // TODO: make container_reduce support tuple of Number and index_t
-        return container_reduce(lengths, math::multiplies{}, Number<1>{});
+        return container_reduce(lengths, math::multiplies{}, Number64<1>{});
```

Comment:
What is return value of "container_reduce"? 32 or 64 bit Number?

Reply:
The return type is the same as the initial value.